app/views/events/_event_list.html.erb

```diff
@@ -47,7 +47,7 @@
 
     <section class="card-back">
       <div class="left">
-        <img class="card-image" src="<%= event.image.url %>" alt="<%= event.name %> image" />
+        <img class="card-image" src="data:image/<%= event.image_content_type %>;base64,<%= event.image_binary %>" alt="<%= event.name %> image" />
```

Comment:
Nice work! You might move this into a helper or partial. I could even see a route specifically to render the image.